user-manager-service/src/main/java/com/crm/food/establishment/user/auth/service/KeyUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.crm.food.establishment.user.auth.service;
+
+import java.security.KeyFactory;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+
+public class KeyUtils {
+
+    public static PrivateKey convertBase64PrivateKeyToJavaPrivateKey(
```

Comment:
I understand that you suggest it from perspective of your experience, but this looks strange for me. 'Cause as it would be in unreadable in case of writing in one line, it remains unreadable in your suggestion